lang/rust/Makefile

```
@@ -0,0 +1,130 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rust
+PKG_VERSION:=1.50.0
+PKG_RELEASE:=1
+
+PKG_LICENSE:=Apache-2.0 MIT
+PKG_LICENSE_FILES:=LICENSE-APACHE LICENSE-MIT
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/rust-lang/rust.git
+PKG_SOURCE_DATE:=2021-02-10
+PKG_SOURCE_VERSION:=cb75ad5db02783e8b0222fee363c5f63f7e2cf5b
+PKG_MIRROR_HASH:=747c6a9f26d143e0ef95f1be4c6ac814e2d5eead1811cba797b04e71f720df02
+PKG_HOST_ONLY:=1
+
+include $(INCLUDE_DIR)/cmake.mk
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/download.mk
+
+HOST_BUILD_DEPENDS:=ninja/host python3/host
+PKG_BUILD_DEPENDS:=rust/host
+CMAKE_INSTALL:=1
+
+# These are environment variables that are used by other packages to
+# define where rustc/cargo are kept.
+CONFIG_HOST_SUFFIX:=$(shell cut -d"-" -f4 <<<"$(GNU_HOST_NAME)")
```

```suggestion
CONFIG_HOST_SUFFIX:=$(word 4,$(subst -, ,$(GNU_HOST_NAME)))
```

Just use built-in functions, it's okay.